fluidsynth/src/sfloader/fluid_defsfont.c

```diff
@@ -3168,9 +3168,9 @@ fixup_sample (SFData * sf)
     {
       sam = (SFSample *) (p->data);
       
-      // The SoundFont 2.4 spec defines the loop start index as the first sample point of the loop
+      /* The SoundFont 2.4 spec defines the loop start index as the first sample point of the loop */
       inv_start = (sam->loopstart < sam->start) || (sam->loopstart >= sam->loopend);
-      // while loop end is the first point AFTER the last sample of the loop.
+      /* while loop end is the first point AFTER the last sample of the loop */
       inv_end = (sam->loopend > sam->end+1) || (sam->loopstart >= sam->loopend);
```

Comment:
Digging in the code I agree. [sam.end comes directly from the sdhr chunk](https://github.com/FluidSynth/fluidsynth/blob/master/fluidsynth/src/sfloader/fluid_defsfont.c#L3073) which specifies it after valid sample data and we dont seem to correct it by -1. But then [fluidsynths documentation is incorrect](https://github.com/FluidSynth/fluidsynth/blob/master/fluidsynth/src/sfloader/fluid_defsfont.h#L102).